src/shared/boardPropertyService.ts

```
@@ -0,0 +1,44 @@
+    }
+    updateBoardProperty(boardId, property, value, params);
+  }
+
+  static deleteBoardProperty(property: string, params: Record<string, any> = {}): void {
+    const boardId = getBoardIdFromURL();
+    if (!boardId) {
+      throw new Error('no board id');
+    }
+    deleteBoardProperty(boardId, property, params);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)